src/app/pages/wallets/my-wallets/my-wallet/pages-wallets-my-wallets-my-wallet.component.ts

```diff
@@ -8,4 +8,10 @@ import { MyWallet } from '../pages-wallets-my-wallet.model';
 })
 export class PagesWalletsMyWalletsMyWalletComponent {
   @Input() public wallet!: MyWallet;
-}
+
+  @Output('walletEdit') public walletEdit = new EventEmitter<MyWallet>();
+
+  public openEditWalletModal(): void {
```

Comment:
This one should be called `editWallet`. This component doesn't know if after emitting even below some other component will open modal or maybe go to another URL or do something else. It is aware only that edit wallet button was clicked.